README.md

```diff
@@ -17,6 +17,7 @@ Follow the OS-specific instructions to install the [Azure Functions Core Tools](
 Install the prerequisites for your desired language:
 
 * [JavaScript](#javascript)
+* [TypeScript](#javascript)
```

Comment:
intentional repeat of `#javascript`?

Reply:
Yeah requirements are the same. I suppose I could rename the section to "Node"